internal/app/trustd/internal/reg/reg.go

```diff
@@ -62,38 +58,3 @@ func (r *Registrator) WriteFile(ctx context.Context, in *proto.WriteFileRequest)
 
 	return resp, nil
 }
-
-// GenerateCertificateFromCSR creates a signed certificate using the provided
-// certificate, key, and CSR.
-func GenerateCertificateFromCSR(crt, key, csr []byte) (signed *x509.Certificate, err error) {
```

Comment:
This is removed since we needed a central place to generate the CSR. We couldn't do that here since it is internal to `trustd`.